lms/djangoapps/hera/utils.py

```diff
@@ -1,16 +1,11 @@
 from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
 from xmodule.modulestore.django import modulestore
 
-
-def get_medal(points):
-    if 90 <= points <= 100:
-        return "platinum"
-    elif 80 <= points <= 89:
-        return "gold"
-    elif 70 <= points <= 79:
-        return "silver"
-    elif 60 <= points <= 69:
-        return "copper"
+try:
```

Comment:
is it possible to relocate the `hera` app to `common/djangoapps` directory?

Reply:
We have some hera lms templates in the edx-theme, do we need to relocate edx-theme templates for correct hera templates mapping?

Reply:
if it's needed and will bring results and won't take a long time - yes).